site/src/pages/CreateWorkspacePage/CreateWorkspacePageExperimental.tsx

```diff
@@ -276,7 +276,7 @@ const CreateWorkspacePageExperimental: FC = () => {
 			<Helmet>
 				<title>{pageTitle(title)}</title>
 			</Helmet>
-			{!latestResponse ||
+			{(!latestResponse && !wsError) ||
```

Comment:
Is this a distinct fix? Also, should we not guard against rendering if either one of these conditions is missing (regardless of whether they're missing together)?

Lastly, this boolean logic is getting difficult to reason about; I wonder if it's time to pull into a distinct boolean.